obspy/io/rg16/__init__.py

```diff
@@ -29,34 +35,40 @@
 
 .. figure:: /_images/rg16_node_orientation.png
 
-Reading
--------
+Reading the waveforms
+---------------------
 The rg16 format can be read using two methods:
 
 1. Using the standard :func:`~obspy.core.stream.read` function. Optionally,
-   the format parameter can be specified as "rg16" for a modest speed-up.
+   the format parameter can be specified as "rg16" for a speed-up.
 
-2. Using the :mod:`obspy.io.rg16` specific function
-   :func:`obspy.io.rg16.core._read_rg16`.
+2. Using the specific function :func:`obspy.io.rg16.core._read_rg16` in the
+   module :mod:`obspy.io.rg16`.
```

Comment:
Using the private function should *not* be advertized as a legit standard use case. I know this has been in there before your PR @thefroid but would you be so kind to remove the ad for the private function? You can have a look at e.g. the mseed landing page: http://docs.obspy.org/master/packages/obspy.io.mseed.html?highlight=_read_mseed

The RG16 landing page should clearly advertize using `read()` and should only mention the private method `_read_rg16()` by stating that additional `kwargs` to pass in to `read()` should be looked up there.

CC @d-chambers